app/javascript/followButtons/initializeFollowButtons.js

```diff
@@ -341,9 +344,11 @@ export const initializeFollowButtons = () => {
     subtree: true,
   });
 
-  InstantClick.on('change', () => {
-    observer.disconnect();
-  });
+  if (typeof instantClick !== 'undefined') {
```

Comment:
Refactor (potentially blocking): Use `getInstantClick`, or is the use case here that sometimes `InstantClick` will never be present vs. it's present but maybe not right away?

https://github.com/forem/forem/blob/7dad856eefe47cf9cac7ea90488b4cc0ada20155/app/javascript/topNavigation/utilities.js#L138-L161

Reply:
oh nice - updated that now 🙂